ocfweb/component/celery.py

```diff
@@ -14,14 +14,13 @@
     broker=settings.CELERY_BROKER,
     backend=settings.CELERY_BACKEND,
 )
-# TODO: use ssl verification
 celery_app.conf.broker_use_ssl = {
-    'ssl_cert_reqs': ssl.CERT_NONE,
+    'ssl_ca_certs': '/etc/ssl/certs/ca-certificates.crt',
+    'ssl_cert_reqs': ssl.CERT_REQUIRED,
 }
-# `redis_backend_use_ssl` is an OCF patch which was proposed upstream:
-# https://github.com/celery/celery/pull/3831
 celery_app.conf.redis_backend_use_ssl = {
-    'ssl_cert_reqs': ssl.CERT_NONE,
+    'ssl_ca_certs': '/etc/ssl/certs/ca-certificates.crt',
+    'ssl_cert_reqs': ssl.CERT_REQUIRED,
```

Comment:
could reduce duplication a bit since these are the same /shrug

```python
celery_app.conf.broker_use_ssl = celery_app.conf.redis_backend_use_ssl = {
    'ssl_ca_certs': '/etc/ssl/certs/ca-certificates.crt',
    'ssl_cert_reqs': ssl.CERT_REQUIRED,
}
```

Reply:
Good point, that is definitely cleaner

Reply:
I'll leave this for now, but I do think this would be better, maybe combined with something in ocflib so that this section doesn't have to repeated across 4 repos and changed in each of them.